src/domain/reservation/manage/filters/ManageReservationsFilters.js

```diff
@@ -136,6 +153,7 @@ class ManageReservationsFilters extends React.Component {
                 />
                 <div className="separator">-</div>
                 <DateField
+                  end
                   id="EndDateField"
                   label={t('ManageReservationsFilters.endDateLabel')}
                   onChange={(value) => {
```

Comment:
make an event handler here only for date selection change, put our logic to that